stroom-util-shared/src/main/java/stroom/util/shared/ErrorMessage.java

```diff
@@ -8,7 +8,7 @@
 
 import java.util.Objects;
 
-@JsonPropertyOrder({"severity", "message"})
+@JsonPropertyOrder({"severity", "message", "node"})
```

Comment:
@JsonPropertyOrder(alphabetic = true)
easier to maintain, unless we really care about the serialised order